### DISCUSSION TITLE
Don't directly use ZOAU "helper" commands

### DISCUSSION THREAD

**Discussion by AnthonyGiorgio** [IBM/zoau-samples#38] [General] [answered]
Hey everyone, just a friendly reminder from the ZOAU dev team!  You should only be using the documented interfaces to the various ZOAU utilities.  This means that if there's a man page for a command that you're using, you're all good.  On the other hand, if you're directly calling the various `*helper` commands, you should adjust your code.  We're planning on removing the "helpers" over time in order to streamline the product, so this might break your code if not addressed.
In short:
:white_check_mark: `dls`, `jsub`, `pcon`, etc.
:x: `dlshelper`, `jsubhelper`, `pconhelper` , etc.

**Comment by ddimatos** [comment, marked as answer]
Thanks Anthony, glad we are doing our best to prepare users.
